docs/applications.rst

```diff
@@ -20,3 +20,18 @@ Streamlink Twitch GUI
 :Info: A NW.js based desktop web application, formerly known as *Livestreamer Twitch GUI*. Browse Twitch.tv and watch
     multiple streams at once. Filter streams by language, receive desktop notifications when followed channels start
     streaming and access the Twitch chat by using customizable chat applications.
+
+
+LiveStreamLinkGUI
+---------------------
+
+.. image:: https://raw.githubusercontent.com/techmouse/livestreamlinkgui/master/screenshot.png
+    :alt: LiveStreamLinkGUI
+
+:Description: A lightweight BASH script
+:Type: Graphical User Interface
+:OS: |Linux| |MacOS| And any other OS that supports BASH, Zenity, and typical \*nix commands
+:Author: `Mouse <https://github.com/techmouse>`_
+:Website: https://github.com/techmouse/livestreamlinkgui
+:Source code: https://github.com/techmouse/livestreamlinkgui
+:Info: A simple GUI for Streamlink, Livestreamer and any future forks. Easily configurable and written in BASH for low resource *nix based media machines. Designed to be compatible with streaming sites, direct links to media (both local and remote), and webpages containing direct links to media. Able to save links, open popout chats in your browser, and generally aims be an all-in-one go-to tool for watching whatever content possible in your favorite video player, instead of your browser.
```

Comment:
We need to escape the `*` on line 37 as well: `*nix based media machines` sorry for missing it.